pyproject.toml

```diff
@@ -0,0 +1,83 @@
+[build-system]
+requires = ["setuptools>=68", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "polyswarm-api"
```

Comment:
If possible, please consider to fix the package name to comply with PEP625: "change hyphen to underline"

The following PS packages requires this here, and will need a bump anyway. So it can be a good time to update them with newer name:
- polyswarm-commons
- polysando
- e2e
- artifact-index
- polyswarm-cli